src/common/extensionMessaging.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as Q from "q";
+import * as http from "http";
+
+/**
+ * Message server parameters.
+ */
+export let ServerParams = {
+    PORT: 8099,
+    HOST: "127.0.0.1"
+};
+
+/**
+ * Defines the messages sent to the extension.
+ * Add new messages to this enum.
+ */
+export enum ExtensionIncomingMessage {
+    START_PACKAGER,
```

Comment:
I'd go with an ExecuteCommand message with a "startPackager", "stopPackager" argument, that can potentially be "runIos" or "runAndroid" too for free if we need to (given that we already have those methods on the commandPaletteHandler). Another alternative to consider, is that we could instead just call a VS Code API to execute a command ( https://code.visualstudio.com/Docs/extensionAPI/vscode-api#commands.executeCommand ). I think that our commands should appear there, so we could potentially use that to reapply breakpoints too when they release it, or any future commands that we add.